library/src/contexts/useSpec.ts

```diff
@@ -2,8 +2,7 @@ import React, { useContext } from 'react';
 import { AsyncAPIDocumentInterface } from '@asyncapi/parser';
 
 export const SpecificationContext =
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-explicit-any
-  React.createContext<AsyncAPIDocumentInterface>(null as any);
+  React.createContext<AsyncAPIDocumentInterface>(null as never);
```

Comment:
I think your changes here is unnecessarily strict and semantically incorrect in this scenario. I'd suggest we use the existing type :)